moveit_planners/trajopt/trajopt_sco/CMakeLists.txt

```diff
@@ -3,17 +3,15 @@ project(trajopt_sco)
 
 add_compile_options(-std=c++11 -Wall -Wextra)
 
-include(ExternalProject)
-
-ExternalProject_Add(osqp
-    PREFIX osqp
-    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=osqp-install
-    # Github tar balls don't code in submodules, so need to
-    # download via git
-    GIT_REPOSITORY https://github.com/oxfordcontrol/osqp.git 
-    GIT_TAG v0.5.0 # currently, not compatible with v0.6.0
-)
-find_package(osqp CONFIG 
+# Download and unpack osqp at configure time
+configure_file(osqp_configure.in
```

Comment:
Since nothing in `osqp_configure.in` is actually changed, maybe use `file(COPY ...)` instead?